src/Components/ImagesTable/ImagesTable.js

```diff
@@ -24,10 +25,11 @@ import {
   Thead,
   Tr,
 } from '@patternfly/react-table';
+import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/redux';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
-import { Link, useNavigate } from 'react-router-dom';
-
+import { Link, useNavigate, useLocation } from 'react-router-dom';
```

Comment:
Ah sorry, I was wrong, the linter complains. There's 1 newline needed between the external and internal imports.